packages/terra-i18n/README.md

```diff
@@ -12,6 +12,7 @@ The terra-i18n component provides the internationalization to the React componen
 
 ## Getting Started
 
+At most time, you don't need to install this package directly. Please follow [this internationalization section](../../README.md#internationalizationi18n) to set up. `Base` will install this package for you.
```

Comment:
We recommend to use `Base` component. Sometimes, consumers want to update add more translations at runtime or support different `locale` under the same `Base`. In those cases, they will want to use I18nProvider directly.

In here, I don't want to point out all those things. It may make consumers get confused. How do you think?